### ISSUE TITLE
[White Label - Markdown]: H1 heading is not defined on the page.

### ISSUE THREAD

**Issue by Kusum199** [microsoft/BotFramework-WebChat#5693]
### What is the PWD impact?

Screen reader users (e.g., those using JAWS, NVDA, or VoiceOver) rely on heading levels to understand page structure and quickly navigate content.

### What browsers and screen readers do this issue affect?

Windows: Edge with Windows Narrator

### Are there any code-based customization done to Web Chat?

No, I am using Web Chat without any customizations except "styleOptions".

### What version of Web Chat are you using?

Latest production

### Which area does this issue affect?

Others or unrelated

### How to reproduce the issue?

1. Open URL: https://compulim.github.io/webchat-loader/ and sign in using valid credentials.
2. Navigate to web chat page and select 4.18.1-main.20240927.ea7a875 (30/9/2024) version and then select mockBot link under present label and then select open web chat in new window link at bottom.
3.Navigate to white label experience chat page and type "Markdown" word in type your message text box.
4.Navigate to the search results.
5.Observe that, H1 heading is not defined on the page.

### What do you expect?

The page should have a single, properly defined <h1> heading that clearly conveys the main topic, appears at the top of the content, follows a logical heading hierarchy, and is accessible to assistive technologies.

### What actually happened?

H1 heading is not defined on the page.

### Do you have any screenshots or recordings to repro the issue?

<img width="957" height="565" alt="Image" src="https://github.com/user-attachments/assets/3aae6065-1813-49b7-8d79-0c2e1b50a1ad" />

### WCAG reference

https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships

### Adaptive Card JSON

```json

```
